packages/stryker/src/isolated-runner/IsolatedRunnerOptions.ts

```diff
@@ -1,7 +1,9 @@
 import { RunnerOptions } from 'stryker-api/test_runner';
+import LoggingClientContext from '../logging/LoggingClientContext';
 
 interface IsolatedRunnerOptions extends RunnerOptions {
   sandboxWorkingFolder: string;
+  loggingContext: LoggingClientContext;
```

Comment:
This loggingContext seems to be very important for everything that is ran in an isolated process. If it's so important why is it not on the runner options itself?

Reply:
I don't understand. What do you mean with "the runner itself"?

It shouldn't be defined on the `RunnerOptions`, as we don't want the test runner plugin to worry about the details of the logging infrastructure, right?

Reply:
I would expect it to be injected as a dependency in the constructor directly instead of through this object.

Reply:
Which constructor? The testrunner constructor doesn't need it. It's a Stryker internal working detail

Reply:
I'm busy fixing this, but will take some more time.